src/client/components/Admin.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+// import './Admin.scss';
+
+class Admin extends React.Component {
+    render() {
+        return (
+            <div id="register-form">
+                <h1>Welcome!</h1>
+                <p>You have been approved as a admin!</p>
+            </div>
+        )
+    };
```

Comment:
no semicolon after class function (though once you turn this into a functional component this one won't matter anymore)